qiskit_nature/problems/second_quantization/lattice/models/fermi_hubbard_model.py

```diff
@@ -29,22 +30,9 @@ def __init__(self, lattice: Lattice, onsite_interaction: complex) -> None:
             lattice: Lattice on which the model is defined.
             onsite_interaction: The strength of the on-site interaction.
         """
-        self._lattice = lattice
+        super().__init__(lattice)
         self.onsite_interaction = onsite_interaction
```

Comment:
I don't see the need for this line. If this is gone, the entire `__init__` can be removed from this subclass.